dimod/vartypes.py

```diff
@@ -81,15 +81,16 @@ def __deepcopy__(self, memo):
 
 
 class Vartype(enum.Enum):
-    """An :py:class:`~enum.Enum` over the types of variables for the binary
-    quadratic model.
+    """An :py:class:`~enum.Enum` over the types of variables for quadratic models.
 
     Attributes:
-        SPIN (:class:`.Vartype`): Vartype for spin-valued models; variables of
-           the model are either -1 or 1.
-        BINARY (:class:`.Vartype`): Vartype for binary models; variables of the
-           model are either 0 or 1.
-
+        SPIN: Vartype for spin-valued binary quadratic models and variables of
+           quadratic models that have values that are either -1 or 1.
+        BINARY: Vartype for binary quadratic models and variables of quadratic
+           models that have values that are either 0 or 1.
+        INTEGER: Vartype for variables in quadratic models
+            that have values of type int.
+        DISCRETE: Alias for ``INTEGER``.
```

Comment:
Let's not mention `DISCRETE`. There is no harm keeping it there in code but it's deprecated for all practical purposes.